html/store.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="/css/store.css">
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="/favicon.ico">
+
+</head>  
+<header>
+    <img src="/img/karma-logo.svg" alt="karma-logo">
+    <nav id="nav" class="nav">
+        <ul class="bar">
+            <li><a class="karma" href="index.html">Meet Karma</a></li>
+            <li><a href="#">How it Works</a></li>
+            <li><a href="#">Store</a></li> 
+            <li><a href="#">Blog</a></li>
+            <li><a href="#">Help</a></li>
+            <li><a href="#">Login</a></li>
+        </ul>
+    </nav>
+</header>
+<body>
```

Comment:
```suggestion
<main>
```
<header> <main> <footer> should all be wrapped in <body> tag